src/Apps/MyCollection/Routes/MyCollectionArtwork/Components/MyCollectionArtworkRequestPriceEstimateSection.tsx

```diff
@@ -1,30 +1,38 @@
-import { Button, Flex, Separator, Text } from "@artsy/palette"
+import { Button, Flex, Text } from "@artsy/palette"
 import { createFragmentContainer, graphql } from "react-relay"
 import { RouterLink } from "System/Router/RouterLink"
-import { Media } from "Utils/Responsive"
 import { MyCollectionArtworkRequestPriceEstimateSection_artwork$data } from "__generated__/MyCollectionArtworkRequestPriceEstimateSection_artwork.graphql"
 import CheckmarkStrokeIcon from "@artsy/icons/CheckmarkStrokeIcon"
 
 interface MyCollectionArtworkRequestPriceEstimateSectionProps {
   artwork: MyCollectionArtworkRequestPriceEstimateSection_artwork$data
+  ctaColor?: "primaryBlack" | "secondaryNeutral" | null
+}
+
+export const MyCollectionPriceEstimateSentSection: React.FC = () => {
+  return (
+    <Flex alignItems="center" flexDirection={"row"} mb={2} mt={2}>
+      <CheckmarkStrokeIcon />
+      <Text variant="sm" ml={0.5}>
+        Price estimate request sent
+      </Text>
+    </Flex>
+  )
 }
 
 export const MyCollectionArtworkRequestPriceEstimateSection: React.FC<MyCollectionArtworkRequestPriceEstimateSectionProps> = ({
   artwork,
+  ctaColor,
```

Comment:
Maybe we can move the default value of this prop here. 🤔
```suggestion
  ctaColor = "primaryBlack",
```